desktopexporter/internal/app/types/ui-types.ts

```diff
@@ -21,24 +21,20 @@ export type SpanWithUIData =
       metadata: SpanUIData;
     };
 
-export type TraceSummaryWithUIData =
-  | {
-      hasRootSpan: true;
-      rootServiceName: string;
-      rootName: string;
-      rootDurationString: string;
-      spanCount: number;
-      traceID: string;
-    }
-  | {
-      hasRootSpan: false;
-      spanCount: number;
-      traceID: string;
-    };
+export type RootSpanWithUIData = {
+  serviceName: string;
+  name: string;
+  durationString: string;
+};
```

Comment:
Consider dropping this. This is no longer significantly different that `RootSpan` and we can likely just calculate the duration in-line in the render